### PR TITLE
refactor: remove google recaptcha

### PR DESCRIPTION
**Pull request by CrhistianT7**
- ReCaptcha removed from contact us component

### CODE REVIEW

File: src/pages/ContactUs/ContactUs.tsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import {
   SectionContactUs,
 } from 'components/ContactUs/ContactUs.styles'
 import ContactUsInformation from 'components/ContactUs/ContactUsInformation/ContactUsInformation'
-import FormContacUs from 'components/ContactUs/FormContactUs/FormContactUs'
+import FormContacUs from 'components/ContactUs/ContactUsForm/ContactUsForm'
```
Comment (Mardecera) [inline src/pages/ContactUs/ContactUs.tsx:8]: Change FormContactUs to ContactUsForm

**Comment by Mardecera** [comment]
Key?

Captura de pantalla 2024-05-17 233228

**Review by GustavoUT22 — APPROVED** [review]
Well done!

**Review by Mardecera — APPROVED** [review]
Nice!

**Pull request merged**